main/src/main/java/cgeo/geocaching/ui/ImageGalleryView.java

```diff
@@ -230,7 +232,7 @@ public void fillView(final ImageGalleryEntryHolder viewHolder, final ImageGaller
             }, () -> {
                 binding.imageProgressBar.setVisibility(View.VISIBLE);
                 binding.imageImage.setImageResource(R.drawable.mark_transparent);
-                //binding.imageImage.setVisibility(View.GONE);
```

Comment:
intentionally left commented-out?

Reply:
Hi @moving-bits I struggle interpreting your comment. As part of the PR I deleted a line which was commented out before anyway (small side-cleanup). What is your request?

Reply:
sorry, my bad. I thought this commented out line was still there, missed that it was marked as deleted in this PR